src/SmartComponents/PatchManager/PatchManagerCard.js

```diff
@@ -22,13 +22,18 @@ import messages from '../../Messages';
 import { useIntl } from 'react-intl';
 
 /**
- * Operating systems card for showing the ratio of operating systems used.
+ * Card for showing the systems and ratios of current advisories.
  */
-const PatchManagerCard = ({ systems, systemsStatus, fetchSystems, fetchSecurity, securityStatus,
-    security, bugs, fetchBugs, bugsStatus, enhancements, fetchEnhancements, enhancementsStatus,
-    selectedTags, workloads, SID }) => {
+const PatchManagerCard = ({
+    systems, systemsStatus, fetchSystems,
+    advisories, advisoriesStatus, fetchAdvisories,
+    selectedTags, workloads, SID
+}) => {
     const intl = useIntl();
-    const isLoaded = [systemsStatus, securityStatus, bugsStatus, enhancementsStatus].every(item => item === 'fulfilled');
+    const isLoaded = [systemsStatus, advisoriesStatus].every(item => item === 'fulfilled');
+    const security = advisories.security;
+    const bugs = advisories.bugfix;
+    const enhancements = advisories.enhancement;
```

Comment:
`advisories` are undefined before the network request is fulfilled. This leads to an error that breaks the whole page component because you are accessing `security` and further parameters on undefined type.

The very first thing came to mind and that fixes this is to provide an alternative value: 
```suggestion
    const security = advisories?.security || 0;
    const bugs = advisories?.bugfix || 0;
    const enhancements = advisories?.enhancement || 0;
```

Reply:
I think this is fixed with `const { security, bugfix: bugs, enhancement: enhancements } = advisories || {};` now?